### ISSUE TITLE
Networking and Programming Languages

### ISSUE THREAD

**Issue by aydencook03** [ForrestKnight/open-source-cs-python#10]
As mentioned in the video, I think "Programming Languages: Part A/B/C" should be included so that people don't get too focused on only a single language.

I also think networking/web-technologies (at least basics) should be included.  It's important to know how the internet works and how computers communicate and share information.

**Comment by BirajMalakar** [comment]
Yeah I feel the same. I think python language and networking should be included.
